deploy/helm/lunettes/templates/grafana/grafana-cm.yaml

```diff
@@ -2609,7 +2609,7 @@ data:
                       {
                         "targetBlank": true,
                         "title": "Pod信息",
-                        "url": "\r\nhttp://lunettes.lunettes.svc.cluster.local:8080/api/v1/debugpod?uid=${__data.fields.PodUid}"
+                        "url": "\r\nd/lunettes-debugpod/lunettes-debugpod?var-podinfo=uid&var-podinfovalue=${__data.fields.PodUid}"
```

Comment:
Can the `\r\n` be removed? This is supposed to be a new line character in Windows, which I don't think is needed here.

Reply:
okay